influxdb/client.py

```diff
@@ -66,8 +67,9 @@ class InfluxDBClient(object):
         as a single file containing the private key and the certificate, or as
         a tuple of both files’ paths, defaults to None
     :type cert: str
-
     :raises ValueError: if cert is provided but ssl is disabled (set to False)
+    :parm gzip: use gzip content encoding to compress requests
```

Comment:
this should be `param`, and should come before the `:raises` block for consistency

Reply:
No problem, thanks!